docs/ops/movement/ShuffleChannels_1.md

```diff
@@ -8,43 +8,57 @@
 
 **Short description**: *ShuffleChannels* permutes data in the channel dimension of the input tensor.
 
+**Detailed description**:
+
+Input ND tensor of `data_shape` is always interpreted as 4D tensor with the following shape:
+
+    dim 0: data_shape[0] * data_shape[1] * ... * data_shape[axis-1] (or 1 if axis == 0)
+    dim 1: group
+    dim 2: data_shape[axis] / group
+    dim 3: data_shape[axis+1] * data_shape[axis+2] * ... * data_shape[data_shape.size()-1] (or 1 if axis points to last dimension)
+
+So the dimensions before and after the dimension pointed by `axis` are flattened and reshaped back to the original shape after channnels shuffling.
```

Comment:
Trailing and leading to `axis` dimensions are flattened...

Reply:
Done